### PR TITLE
fix #347: add docs_zh_CN/datasets.md

### LINKED ISSUE #347: [Need help] [中文文档翻译] Chinese documentation
Hi all,

We are asking for help from our community, hoping to provide Chinese documentation to all users. It's not difficult and here is the chance to be one of the contributors of MMOCR, an open-source rising star!

Here is a list of documentation needed to be translated:

- [ ] [docs/getting_started.md](https://github.com/open-mmlab/mmocr/blob/main/docs/getting_started.md)
- [x] [docs/install.md](https://github.com/open-mmlab/mmocr/blob/main/docs/install.md) (@A465539338)
- [x] [docs/datasets.md](https://github.com/open-mmlab/mmocr/blob/main/docs/datasets.md) (@doem97)
- [x] [docs/deployment.md](https://github.com/open-mmlab/mmocr/blob/main/docs/deployment.md) (@fatfishZhao)
- [ ] [demo/MMOCR_Tutorial.ipynb](https://github.com/open-mmlab/mmocr/blob/main/demo/MMOCR_Tutorial.ipynb) 
- [ ] [docs/demo.md](https://github.com/open-mmlab/mmocr/blob/main/demo/docs/ocr_demo.md)

Interested users can directly reply to this issue and tell us the documentation you would like to translate. Then, please read our guidence (will be announced later) before you finalize your translation and make the PR. Thank you for your interests and we really appreciate your contribution :)

# Guidence 
All the docs needed to be translated are now put in `docs_zh_CN/`. Upon finished, just create a PR and wait to see how your contribution makes the change. Additional info about the contribution process can be found at https://zhuanlan.zhihu.com/p/387116301 (written in Chinese).

Note: Some examples can be found in `demo/docs_zh_CN/`.

### PR DESCRIPTION
**Pull request by doem97**
Add CN version of `datasets` document.

### CODE REVIEW

File: docs_zh_CN/datasets.md [hunk 1]
```diff
@@ -259,15 +264,15 @@ The structure of the text detection dataset directory is organized as follows.
 
   mv /path/to/shuffle_labels.txt .
 
-  # create soft link
+  # 创建soft link
```
Comment (gaotongxiao) [inline docs_zh_CN/datasets.md:267]: 创建软链接

File: docs_zh_CN/datasets.md [hunk 2]
```diff
@@ -277,23 +282,23 @@ The structure of the text detection dataset directory is organized as follows.
...
```
Comment (gaotongxiao) [inline docs_zh_CN/datasets.md:285]: 创建软链接

File: docs_zh_CN/datasets.md [hunk 3]
```diff
@@ -304,69 +309,67 @@ The structure of the text detection dataset directory is organized as follows.
 
   mv /path/to/label.txt .
 
-  # create soft link
+  # 创建 soft link
```
Comment (gaotongxiao) [inline docs_zh_CN/datasets.md:312]: 创建软链接

File: docs_zh_CN/datasets.md [hunk 4]
```diff
@@ -304,69 +309,67 @@ The structure of the text detection dataset directory is organized as follows.
...
   cd /path/to/mmocr/data/mixture
 
   ln -s /path/to/SynthAdd SynthAdd
   ```
-  **Note:**
-To convert label file with `txt` format to `lmdb` format,
+  **注意：**
+把 `.txt` 格式的标注文件转换成 `.lmdb` 格式：
```
Comment (gaotongxiao) [inline docs_zh_CN/datasets.md:318]: **额外说明：**
运行以下命令，可以把 `.txt` 格式的标注文件转换成 `.lmdb` 格式：

File: docs_zh_CN/datasets.md [hunk 5]
```diff
@@ -304,69 +309,67 @@ The structure of the text detection dataset directory is organized as follows.
...
   wget https://dl.fbaipublicfiles.com/textvqa/data/textocr/TextOCR_0.1_train.json
   wget https://dl.fbaipublicfiles.com/textvqa/data/textocr/TextOCR_0.1_val.json
 
-  # For images
+  # 对于数据图像
   unzip -q train_val_images.zip
   mv train_images train
   ```
-  - Step2: Generate `train_label.txt`, `val_label.txt` and crop images using 4 processes with the following command:
+  - 第二步：用四个并行线程剪裁图像然后生成  `train_label.txt`，`val_label.txt` ，可以使用以下命令：
```
Comment (gaotongxiao) [inline docs_zh_CN/datasets.md:340]: 线程 -> 进程

**Review by gaotongxiao — COMMENTED** [review]
Nice work! I've left few translation suggestions, and you might change them if you agree that's better.

**Comment by doem97** [comment]
> Nice work! I've left few translation suggestions, and you might change them if you agree that's better.

Thanks for the accurate corrections. Agree with you.

Besides, it's strange that `CLUENER2020` is a separate section in TOC, while other datasets are just listed items. Is it intended to do so?

**Comment by gaotongxiao** [comment]
> Besides, it's strange that `CLUENER2020` is a separate section in TOC, while other datasets are just listed items. Is it intended to do so?

It seems really unnecessary. Thanks for pointing that out!

**Review by gaotongxiao — APPROVED** [review]

**Comment by OpenMMLab-Coordinator** [comment]
doem97，您好！您在MMOCR项目中给我们提的PR非常重要，感谢您付出私人时间帮助改进开源项目，相信很多开发者会从你的PR中受益。
我们非常期待与您继续合作，OpenMMLab专门成立了贡献者组织MMSIG，为贡献者们提供开源证书、荣誉体系和专享好礼，可通过添加微信：openmmlabwx 联系我们（请备注mmsig+GitHub id），由衷希望您能加入！


Hi @doem97 ！First of all, we want to express our gratitude for your significant PR in the MMOCR project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/raweFPmdzG

If you have WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request merged**